Script.php

```diff
@@ -1086,30 +1068,62 @@ function checksize($file)
    
    
    
-       #Create Group Request
+       //---------------------------------------Invite Group Request and add a new member into the database------------------------------------
   
  if (!empty($_GET["groupinvite"])) {
 	   
 	   $student_id=$_GET["student_id"];
 	    $url=$_GET["url"];
                $courseid=$_GET["courseid"];
                 $groupid=$_GET["groupid"];
-            
-           
-                
-                   
-                
-   
+               
+  if(($_SESSION['Group_Member4']!='0') or ($_SESSION['Group_Member3']!='0') or ($_SESSION['Group_Member2']!='0') or ($_SESSION['Group_Member']!='0')){
              $sql="INSERT INTO `course_group_members_table`( `Course_Group_id`, `Student_ID`, `Status`) 
                           VALUES ($groupid,$student_id,'Invited')";
    if ($con->query($sql) === TRUE) {
-         $_SESSION["info_ReMarking"]=$student_id . " was invited to the group";
-          header("Location: Course.php?url=".$url); 
-   } else {
-    echo "Error: " . $sql . "<br>" . $con->error;
-}
+
+    $resultx1 = mysqli_query($con,"SELECT * FROM course_groups_table where Course_Group_id ='$groupid'");
+   
+    while($row = mysqli_fetch_assoc($resultx1)) 
+    {
+      $Group_Member=$row['Group_Member']; 
+      $Group_Member4=$row['Group_Member4'];
+      $Group_Member2=$row['Group_Member2'];
+      $Group_Member3=$row['Group_Member3'];
+      $_SESSION['Group_Member4']=$Group_Member4;
+      $_SESSION['Group_Member3']=$Group_Member3;
+      $_SESSION['Group_Member2']=$Group_Member2;
+      $_SESSION['Group_Member']=$Group_Member;
```

Comment:
@Teecloudy '

Better line order.  Member, Member2, Member3, and Member4 should appear in a proper order.

**Hui**

Reply:
Noted